Source/CarthageKit/Xcode.swift

```diff
@@ -858,21 +863,33 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 	}
 
 	return BuildSettings.SDKsForScheme(scheme, inProject: project)
-		|> map { $0.platform }
-		|> flatMap(.Concat) { (platform: Platform) in
+		|> collect
+		|> flatMap(.Concat) { (schemeSDKList: [SDK]) in
+			let platform: Platform! = schemeSDKList.first!.platform
```

Comment:
If `schemeSDKList` is empty, unwrapping `first` would crash, so the following `fatalError()` message would not be printed, I think. Using IUO is OK, so we should avoid unwrapping:

```swift
let platform: Platform! = schemeSDKList.first?.platform
```